From 466277e6d23b20c688014c77fba3e6d1686bf8c5 Mon Sep 17 00:00:00 2001
From: Artem Lapkin <art@khadas.com>
Date: Fri, 25 Sep 2020 15:19:01 +0800
Subject: [PATCH] arm64: dts: meson: rename led label for VIM2
 meson-gxm-khadas-vim2

rename led label for VIM2 meson-gxm-khadas-vim2

SYS_LED label renamed from vim:red:power to vim2:white:sys

Signed-off-by: Artem Lapkin <art@khadas.com>
---
 arch/arm64/boot/dts/amlogic/meson-gxm-khadas-vim2.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/amlogic/meson-gxm-khadas-vim2.dts b/arch/arm64/boot/dts/amlogic/meson-gxm-khadas-vim2.dts
index 76b7e34a9a3..7572c132d2f 100644
--- a/arch/arm64/boot/dts/amlogic/meson-gxm-khadas-vim2.dts
+++ b/arch/arm64/boot/dts/amlogic/meson-gxm-khadas-vim2.dts
@@ -85,7 +85,7 @@ pwmleds {
 		compatible = "pwm-leds";
 
 		power {
-			label = "vim:red:power";
+			label = "vim2:white:sys";
 			pwms = <&pwm_AO_ab 1 7812500 0>;
 			max-brightness = <255>;
 			linux,default-trigger = "default-on";
-- 
2.25.1

